src/main/java/kr/syeyoung/dungeonsguide/commands/CommandDgDebug.java

```diff
@@ -0,0 +1,525 @@
+                                    }
+                                }
+                            } else if (value instanceof DungeonDoor) {
+                                for (OffsetPoint offsetPoint : ((DungeonDoor) value).getSecretPoint().getOffsetPointList()) {
+                                    if (offsetPoint.getY() == 0 && dri.getBlocks()[offsetPoint.getZ()][offsetPoint.getX()] != -1) {
+                                        dri.getBlocks()[offsetPoint.getZ()][offsetPoint.getX()] = -1;
+                                        System.out.println("Fixing " + value2.getKey() + " - door - at " + offsetPoint);
+                                    }
+                                }
+                            } else if (value instanceof DungeonBreakableWall) {
+                                for (OffsetPoint offsetPoint : ((DungeonBreakableWall) value).getSecretPoint().getOffsetPointList()) {
+                                    if (offsetPoint.getY() == 0 && dri.getBlocks()[offsetPoint.getZ()][offsetPoint.getX()] != -1) {
+                                        dri.getBlocks()[offsetPoint.getZ()][offsetPoint.getX()] = -1;
+                                        System.out.println("Fixing " + value2.getKey() + " - wall - at " + offsetPoint);
+                                    }
+                                }
+                            } else if (value instanceof DungeonTomb) {
+                                for (OffsetPoint offsetPoint : ((DungeonTomb) value).getSecretPoint().getOffsetPointList()) {
+                                    if (offsetPoint.getY() == 0 && dri.getBlocks()[offsetPoint.getZ()][offsetPoint.getX()] != -1) {
+                                        dri.getBlocks()[offsetPoint.getZ()][offsetPoint.getX()] = -1;
+                                        System.out.println("Fixing " + value2.getKey() + " - crypt - at " + offsetPoint);
+                                    }
+                                }
+                            }
+                        }
+                        FileOutputStream fos = new FileOutputStream(f);
+                        ObjectOutputStream oos = new ObjectOutputStream(fos);
+                        oos.writeObject(dri);
+                        oos.flush();
+                        oos.close();
+                    } catch (Exception e) {
+                        e.printStackTrace();
+                    }
+                }
+                break;
+            case "reloaddungeon":
+                try {
+                    MinecraftForge.EVENT_BUS.post(new DungeonLeftEvent());
+                    DungeonsGuide.getDungeonsGuide().getSkyblockStatus().setContext(null);
+                    MapUtils.clearMap();
+                } catch (Throwable t) {
+                    t.printStackTrace();
+                }
+                break;
+            case "partymax":
```

Comment:
Why is partymax a debug command? PartyMax is not supposed to be a debug command. It's for users to change how many people can join their party via the discord thingy breh